tests/unit/blog-utils/TestStaticAddonCard.js

```diff
@@ -62,9 +62,6 @@ describe(__filename, () => {
 
     const root = render({ addon });
 
-    expect(root.find('.StaticAddonCard-summary').html()).not.toContain(
```

Comment:
(I am thinking that these changes in `tests/unit/*` could land ahead of this PR because they will still be useful)

Reply:
This is a good idea. I can extract them into a separate issue/PR.

Reply:
I think this is actually good to merge now, so I'm not sure it's worthwhile extracting these changes. 

@willdurand If you're fine with this PR now please approve it. If you think it's important that the test changes be extracted let me know and I'll do that in a separate patch.